go-controller/pkg/allocator/pod/macs.go

```diff
@@ -0,0 +1,21 @@
+package pod
+
+import (
+	"fmt"
+
+	kubevirtv1 "kubevirt.io/api/core/v1"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+// GetMACOwner compose the owner identifier reserved for MAC addresses management.
+// Returns "<ns>/<pod-name>" for regular pods and "<ns>/<vm-name>" for VMs.
+func GetMACOwner(pod *corev1.Pod) string {
```

Comment:
asking now so I don't forget - but can't this be a private function?

Reply:
I added unit test for this function so it should remain exported for testing.

As for the added tests I couldn't find another place to test this logic, feel free to suggest alternatives.

Reply:
Done.
Tests removed and covered by higher layer unit-tests, and the functions is unexported.